db/migrate/20161010113526_migrate_reason_to_reasons.rb

```diff
@@ -0,0 +1,7 @@
+class MigrateReasonToReasons < ActiveRecord::Migration
+  def change
+    Rejection::REASONS.each do |reason|
```

Comment:
I think it would be safer to put this on a rake task because of the type of postgres locks that migrations use. Unless we deploy this out of hours when we are sure that staff aren't using the dashboards.

It would also be worth checking how big the batches are as updating large batches could cause load on the db to go up. If the batches can be very large then we could update it in batches of 10k with sleeps in between.